Taskfile.yaml

```diff
@@ -151,12 +151,13 @@ tasks:
     vars:
       PYLINT_DISABLE_FOR_SRCS: 'too-few-public-methods,logging-fstring-interpolation,useless-parent-delegation'
       PYLINT_DISABLE_FOR_TESTS: 'redefined-outer-name,invalid-name,protected-access,too-few-public-methods,unspecified-encoding,duplicate-code'
+      PYLINT_PLUGINS: 'pylint_pydantic'
     sources:
       - featurebyte/**/*
       - tests/**/*
     cmds:
-      - poetry run pylint --disable={{ .PYLINT_DISABLE_FOR_SRCS }} --rcfile pyproject.toml featurebyte
-      - poetry run pylint --disable={{ .PYLINT_DISABLE_FOR_TESTS }} --rcfile pyproject.toml tests
+      - poetry run pylint --load-plugins {{ .PYLINT_PLUGINS }} --disable={{ .PYLINT_DISABLE_FOR_SRCS }} --rcfile pyproject.toml featurebyte
+      - poetry run pylint --load-plugins {{ .PYLINT_PLUGINS }} --disable={{ .PYLINT_DISABLE_FOR_TESTS }} --rcfile pyproject.toml tests
```

Comment:
Introduce `pylint_pydantic` plugin to handle pylint `no-member` errors when refering attribute of pydantic model.